cmd/pageship/app/generate.go

```diff
@@ -0,0 +1,80 @@
+package app
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/oursky/pageship/internal/config"
+	"github.com/pelletier/go-toml/v2"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(generateCmd)
+	generateCmd.AddCommand(generateDockerfileCmd)
+}
+
+var generateCmd = &cobra.Command{
+	Use:   "generate [command]",
+	Short: "Generate files",
+	Args:  cobra.NoArgs, //if unknown command, will return error just like main pageship command
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cmd.Help() //show help if no subcommand supplied
+		return nil
+	},
+}
+
+var Version = "dev"
+var dockerfileTemplate = `FROM ghcr.io/oursky/pageship:v%s
```

Comment:
Please use the built-in template library (https://pkg.go.dev/text/template) for text templates.